lib/thirdparty/LBRY.class.php

```diff
@@ -25,6 +25,21 @@ public static function subscribe($email, $tag = null)
     ]), ['json_response' => true]);
     }
 
+    public static function editEmailSettings($token, $email, $isPrimary =null, $isEnabled = null)
+    {
+        return Curl::post(static::getApiUrl('/user/email/edit'),['auth_token' => $token],['email' => $email],['is_primary' => $isPrimary],['is_enabled' => $isEnabled]);
```

Comment:
Shouldn't the last 3 parameters to this call be wrapped in an array? Same for the calls below this, too.